integrations/huggingface-transformers/tutorials/text-classification/text-classification-cli.md

```diff
@@ -0,0 +1,583 @@
+      exclude_module_types: ['LayerNorm', 'Tanh']
+      submodules:
+        - bert.embeddings
+        - bert.encoder
+        - bert.pooler
+        - classifier
+
+distillation_modifiers:
+  - !DistillationModifier
+     hardness: eval(distill_hardness)
+     temperature: eval(distill_temperature)
+     distill_output_keys: [logits]
+
+constant_modifiers:
+  - !ConstantPruningModifier
+      start_epoch: 0.0
+      params: __ALL_PRUNABLE__
+```
+
+</details>
+
+You can download the recipe with the following code:
+
+```python
+from sparsezoo import Model
+transfer_stub = "zoo:nlp/text_classification/obert-base/pytorch/huggingface/qqp/pruned90_quant-none"
+download_dir = "./transfer_recipe-qqp"
+zoo_model = Model(transfer_stub, download_path=download_dir)
+recipe_path = zoo_model.recipes.default.path
+print(recipe_path)
+```
+
+### **Export to ONNX**
+
+Once you have trained your model, export to ONNX in order to deploy with DeepSparse. The artifacts of the training process 
+are saved to your local filesystem. 
+
+Run the following to convert your PyTorch checkpoint to ONNX:
+
+```bash
+sparseml.transformers.export_onnx \
+  --model_path obert_base_pruned90_quant_qqp \
+  --task text_classification
+```
+
+A `deployment` folder is created in your local directory, which has all of the files needed for deployment with DeepSparse including the `model.onnx`, `config.json`, and `tokenizer.json` files.
+
+## Sparse Transfer Learning with a Single Input Custom Dataset (TweetEval)
+
+Beyond the built-in GLUE tasks, we can also use datasets for single input multi-class classification problems. The datasets can either be passed as Hugging Face hub model identifiers or via local CSV/JSON files.
+
+Let's try to transfer onto the [TweetEval Emotion Dataset](https://huggingface.co/datasets/tweet_eval). This dataset 
+contains single sentences with 4 labels representing the emotion of the tweet (`0=anger, 1=joy, 2=optimism, 3=sadness`.
+
+For simplicity, we will perform the fine-tuning without distillation. Although the transfer learning recipe contains distillation modifiers, we can turn them off by setting `--distill_teacher disable`.
+
+### Inspecting TweetEval Dataset
+
+Run the following to inspect the TweetEval
+
+```python
+from datasets import load_dataset
+from pprint import pprint
+
+emotion = load_dataset("tweet_eval", "emotion")
+print(emotion)
+pprint(emotion["train"][0])
+```
+
+We can see that each row dataset contains a `text` field which is a string representing the sequence to be classified and a `label` field which is in `{0,1,2,3}` representing
+one of four emotions.
+
+### Using a Hugging Face Dataset Identifier
+
+We can easily pass the `tweet_eval` Hugging Face dataset identifier to the script. Simply replace the `--task_name` argument with `--dataset_name tweet_eval --dataset_config_name emotion --input_column_names text --label_column_name label`. Since the TweetEval dataset contains multiple subsets (e.g. there is a subset
```

Comment:
Since the TweetEval dataset contains multiple subsets (e.g., a subset that classifies text into an emoji), we pass the `--dataset_config_name` to specify the subset.